s3/replication/common/tests/system/test_head_object.py

```diff
@@ -50,10 +50,16 @@ async def main():
     bucket_name = config.source_bucket_name
     request_id = "dummy-request-id"
 
+    # For Cortx, Default value of version id=None
+    # For AWS, please provide the version id
```

Comment:
# For AWS, user of this class should provide version-id as well.

Please change the comment.